packages/design/jobberStyle.js

```diff
@@ -248,3 +248,62 @@ function removeNewLines(text) {
   if (!text) return text;
   return text.replace(/(\r\n|\n|\r)/gm, "");
 }
+
+function writeMobileFoundationFiles() {
+  const { androidShadows, iOSShadows } = getShadowStyles(resolvedCssVars);
+  const androidFoundationJobberStyle = {
+    ...resolvedCssVars,
+    ...androidShadows,
+  };
+  const iOSFoundationJobberStyle = {
+    ...resolvedCssVars,
+    ...iOSShadows,
+  };
+  const androidFoundationsExportString = `export const JobberStyle = ${JSON.stringify(
+    androidFoundationJobberStyle,
+    undefined,
+    2,
+  )}`;
+
+  const iOSFoundationsExportString = `export const JobberStyle = ${JSON.stringify(
+    iOSFoundationJobberStyle,
+    undefined,
+    2,
+  )}`;
+  fs.writeFile(
+    "./foundation.android.js",
+    androidFoundationsExportString,
+    "utf-8",
+    err => {
+      if (err) {
+        console.log("An error occured while writing Android Foundation File.");
+        return console.log(err);
+      }
+      console.log("Wrote Android Foundations file");
+    },
+  );
+  fs.writeFile(
+    "./foundation.ios.js",
+    iOSFoundationsExportString,
+    "utf-8",
+    err => {
+      if (err) {
+        console.log("An error occured while writing iOS Foundation File.");
+        return console.log(err);
+      }
+      console.log("Wrote iOS Foundations file");
+    },
+  );
+  fs.writeFile(
+    "./foundation.native.js",
```

Comment:
I create a `foundation.native.js` and corresponding types to play nicely with typescript in react native projects such as `jobber-mobile`. In the readme I give an explanation as to why (the tldr is typescript doesn't resolve platform specific files for type definitions)